.env.development

```diff
@@ -7,5 +7,6 @@ REACT_APP_REPORTING_API_ROOT=https://dev-k8s.treetracker.org/reporting
 REACT_APP_TREETRACKER_API_ROOT=https://dev-k8s.treetracker.org/treetracker
 REACT_APP_ENABLE_CAPTURE_MATCHING=true
 REACT_APP_ENABLE_EARNINGS=true
-REACT_APP_ENABLE_PAYMENTS=true
 REACT_APP_ENABLE_MESSAGING=true
+REACT_APP_ENABLE_PAYMENTS=true
+REACT_APP_REPORTING_ENABLED=false
```

Comment:
Why is REPORTING marked as false here?  I believe reporting should always be true now, as it is deployed into production.